Version one

RoyNyaga opened the pull request:
There are some comments i left in my html and css files, it's for the purpose of easy review. Will clean that up when the project is accepted.

RoyNyaga committed:
continued with css

RoyNyaga committed:
added photos and ajusted responsiveness

RoyNyaga committed:
added stickler file

Ispirett commented:
Hey, @RoyNyaga  great work with this project you did a  fantastic job :1st_place_medal: Also love your read me file :+1: .  Just some minor fixes 

### HTML errors
There are a  few  HTML errors on this page use this tool and please correct them https://validator.w3.org/.

**When finished you can submit for review.**

### Optional changes
Using `<section>` tags instead of `<div>` for groups of related articles is good practice.
`<.list-links >` for margin-right consider using shorthand syntax like `magin: 10px 10px 10px 10px `            and padding-right is overidering another decaration high up in the that class consider removing one.


You can consider adding this section to give your site to get that complete pristine look :100:  
![bootstrap](https://user-images.githubusercontent.com/38801423/62511878-aa0dc880-b7e3-11e9-926e-e3c8dd51a645.png)

RoyNyaga committed:
Used section tags instead of div for sections in the main, added the last section of the main, validated markup for errors

RoyNyaga commented:
I rectified all the errors thrown by the markup validator and implemented section tags for main sections in the main tag. I also added the last section of the main tag as requested. I purposely left comments in my work for easy review in case there are any.

bolah2009 commented:
Currently under review

bolah2009 commented on css/style.css:
```diff
@@ -0,0 +1,461 @@
+  background-color: white;
+}
+.header-nav::before {
+  position: absolute;
```
`SUGGESTION`

Consider using a bootstrap class instead. _Note that the project is about using bootstrap._

bolah2009 commented on css/style.css:
```diff
@@ -0,0 +1,461 @@
+}
+
+.header-article {
+  position: relative;
```
`SUGGESTION`

Consider using a bootstrap class instead. _Note that the project is about using bootstrap._

bolah2009 commented on css/style.css:
```diff
@@ -0,0 +1,461 @@
+}
+.news-btn::after {
+  content: "";
+  display: inline-block;
```
`SUGGESTION`

Consider using a bootstrap class instead. _Note that the project is about using bootstrap._

bolah2009 reviewed (changes requested):
## Awesome Job [Roy](../../) :clap: 

Awesome job so far, the [clone](https://raw.githack.com/RoyNyaga/using_bootstrap/version_one/index.html) looks good to me and very responsive.. :clap:

However, use more of bootstrap as pointed out in [this comment](https://github.com/RoyNyaga/using_bootstrap/pull/1#discussion_r311328999) and all other occurrences and make the navbar fixed as in the [original site](https://www.newsweek.com/), this is to make the homepage and website menu accessibility anyway in the page especially for long websites which is the case in this website. See below for illustration:

|Current|Preffered|
| :-----: | :------: |
| ![newsweek fix navbar cureent](https://user-images.githubusercontent.com/36057474/62586315-f2e18200-b8b4-11e9-82c5-7e631e42e93e.gif) | ![newsweek fix navbar preffred](https://user-images.githubusercontent.com/36057474/62586327-fbd25380-b8b4-11e9-92bd-7247ab96a9ca.gif) |

Good luck with your next submission.

Cheers
[Bola](https://bolabuari.com/)

RoyNyaga committed:
used the bootstrap navbar fixed to make a fixed nav and included more bootstrap classes for background and margin

RoyNyaga commented:
Added the bootstrap navbar-fixed to create a fixed nav and added more bootstrap classes as demanded. Notice in some areas i couldn't use the bootstrap margin and padding because the highest it could produce (ie m-5) was not enough to produce the effect i wanted. Bootstrap colors are a little bit limited to their default colors, so i had to use CSS colors in some areas.

RoyNyaga committed:
Ajusted the red color of the subnav

samgaco commented:
### Hi @RoyNyaga :hand: 

### Good work implementing the changes required in the previous reviews. Congratulations because your project is approved :+1: :tada: 

**[OPTIONAL]** _There are some things that you could still improve like the icons at the footer (they should arrange below the logo in the smaller screen size, or the opinion section in the medium screen size get's too much empty space at its right. Those would be things to work on if you would like to keep digging into this project._

### You may merge to the master branch and submit the project completion form. Keep the good work for the next projects :muscle: Regards!.

RoyNyaga commented:
Ok thanks for the quick review, i will try and improve the icons at the
footer for smaller screens before submitting.

On Wed, Aug 7, 2019 at 8:23 PM samgaco <notifications@github.com> wrote:

> Hi @RoyNyaga <https://github.com/RoyNyaga> ✋ Good work implementing the
> changes required in the previous reviews. Congratulations because your
> project is approved 👍 🎉
>
> *[OPTIONAL]* *There are some things that you could still improve like the
> icons at the footer (they should arrange below the logo in the smaller
> screen size, or the opinion section in the medium screen size get's too
> much empty space at its right. Those would be things to work on if you
> would like to keep digging into this project.*
> You may merge to the master branch and submit the project completion form.
> Keep the good work for the next projects 💪 Regards!.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/RoyNyaga/using_bootstrap/pull/1?email_source=notifications&email_token=ALZGFHNWLFQR4ALVJAQ6EO3QDMOJNA5CNFSM4IJRLJUKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD3ZONSI#issuecomment-519235273>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALZGFHINQEQQP43NGJ36WXDQDMOJNANCNFSM4IJRLJUA>
> .
>
